lib/api/src/sys/tunables.rs

```diff
@@ -120,6 +120,16 @@ mod tests {
         fn try_clone(&self) -> Option<Box<dyn LinearMemory + 'static>> {
             None
         }
+        fn fork(&mut self) -> Result<Box<dyn LinearMemory + 'static>, MemoryError> {
```

Comment:
Let's use clone. I think is a more accurate naming

Reply:
Renamed to `duplicate()`.